### DISCUSSION TITLE
Seal works perfectly with WSA (Windows Subsystem for Android)

### DISCUSSION THREAD

**Discussion by BobbyESP** [JunkFood02/Seal#210] [Show and tell]
I just wanted to tell you guys that Seal works really nice in Windows Subsystem for Windows. I used the [BuildWSA](https://github.com/Paxxs/BuildWSA) repo to install the subsystem and i've downloaded Seal with the release of this repo.

<img width="514" alt="image" src="https://user-images.githubusercontent.com/60316747/189481660-8aef93de-d34c-4eef-9f8a-5562c53cdb0e.png">

https://user-images.githubusercontent.com/60316747/189481702-805c605f-5ee3-4e5a-8cbb-05ca2962b808.mp4

**Comment by JunkFood02** [comment]
Looks pretty nice! I'll give it a try later with WSA too

**Reply by BobbyESP** [reply]
Okayy!

**Comment by BobbyESP** [comment]
It's impossible to port from Jetpack Compose to Jetpack Compose Desktop, it isn't?
